test/integration_test.py

```diff
@@ -101,6 +102,19 @@ def test_sac_system(fast_config: dict, system_path: str) -> None:
     _run_system(system_path, cfg)
 
 
+@pytest.mark.parametrize("system_path", transformer_systems)
+def test_transformer_system(fast_config: dict, system_path: str) -> None:
+    """Test transformer systems on random envs."""
+    _, _, system_name = system_path.split(".")
+    env = random.choice(continuous_envs + discrete_envs)
+
+    with initialize(version_base=None, config_path=config_path):
+        cfg = compose(config_name=f"{system_name}", overrides=[f"env={env}"])
+        cfg = _get_fast_config(cfg, fast_config)
+
+    _run_system(system_path, cfg)
+
+
 @pytest.mark.parametrize("env_name", discrete_envs)
 def test_discrete_env(fast_config: dict, env_name: str) -> None:
     """Test all discrete envs on random systems."""
```

Comment:
if you can MAT to the random choice here

Reply:
I spoke to Sasha about this and sicne goal is to just test the envs, it makes more sense to omit MAT here, since it will be the slowest. I think we should do the same for Sable where we only test the algo but exclude it from the env tests.

Reply:
Sure, i will update sable PR based on this